technique/muon/muon_begin_end.py

```diff
@@ -0,0 +1,271 @@
+        The sample parameters that is to contain the orientation parameter.
+    """
+    old_orientation = sample_pars.get(orient_par, "")
+    new_orient = get_input("Orientation {}?".format(old_orientation))
+    if new_orient != "":
+        g.change_sample_par(orient_par, new_orient)
+
+
+def set_temp_sample_par(sample_pars):
+    """
+    Ask the user if the sample parameter (temp) is correct and allow them to change it if not.
+
+    Parameters
+    ----------
+    sample_pars: dict
+        The sample parameters that is to contain the temp parameter.
+    """
+    old_temp = sample_pars.get(temp_par, "")
+    new_temp = get_input("Temperature {}?".format(old_temp))
+    if new_temp != "":
+        g.change_sample_par(temp_par, new_temp)
+
+
+def set_field_sample_par(sample_pars):
+    """
+    Ask the user if the sample parameter (field) is correct and allow them to change it if not.
+
+    Parameters
+    ----------
+    sample_pars: dict
+        The sample parameters that is to contain the field parameter.
+    """
+    old_field = sample_pars.get(field_par)
+    new_field = get_input("Field {}?".format(old_field))
+    if new_field != "":
+        g.change_sample_par(field_par, new_field)
+
+
+def set_geometry_beamline_par(beamline_pars):
+    """
+    Ask the user if the sample parameter (geometry) is correct and allow them to change it if not.
+
+    Parameters
+    ----------
+    beamline_pars: dict
+        The sample parameters that is to contain the geometry parameter.
+    """
+    old_geometry = beamline_pars.get(geometry_par, "")
+    new_geometry = None
+    while new_geometry not in {"", "L", "T"}:
+        new_geometry = get_input("Geometry (L or T) {}?".format(old_geometry))
+    if new_geometry != "":
+        g.change_beamline_par(geometry_par, new_geometry)
+
+
+def set_rb_num():
+    """
+    Ask the user if the rb number is correct and allow them to change it if not.
+    """
+    old_rb_num = g.get_rb()
+    while True:
+        new_rb_num = get_input("RBNo {}?".format(old_rb_num))
+        if new_rb_num == "" or new_rb_num.isdigit():
+            break
+        else:
+            print("RB number must be a number or blank to skip setting")
+    if new_rb_num != "":
+        g.change_rb(new_rb_num)
+
+
+def set_users():
+    """
+    Ask the user if the users are correct and allow them to change it if not.
+    """
+    old_exp_team = g.get_users()
+    new_exp_team = get_input("Experimental Team {}?".format(old_exp_team))
+    if new_exp_team != "":
+        g.change_users(new_exp_team)
+
+
+def set_comments_sample_par(sample_pars):
+    """
+    Ask the user if the sample parameter (comments) is correct and allow them to change it if not.
+
+    Parameters
+    ----------
+    sample_pars: dict
+        The sample parameters that is to contain the comments parameter.
+    """
+    if comment_par in sample_pars:
+        old_comments = sample_pars[comment_par]
+    else:
+        old_comments = ""
```

Comment:
As above can use `get`

Reply:
Thank you for picking that up, missed it!